data/json/recipes/ammo/pistol.json

```diff
@@ -1,5 +1,41 @@
 [
   {
+    "result": "36navy",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_PISTOL",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "skills_required": [ "gun", 1 ],
+    "time": 15000,
+    "book_learn": [ [ "manual_pistol", 3 ], [ "recipe_bullets", 3 ] ],
+    "using": [ [ "ammo_bullet", 30 ] ],
+    "qualities": [ { "id": "CUT", "level": 1, "amount": 1 } ],
+    "components": [
+      [ [ "gunpowder", 100 ], [ "chem_black_powder", 200 ] ],
+      [ [ "smpistol_primer", 25 ] ],
+      [ [ "bag_plastic", 1 ], [ "paper", 12 ] ]
+    ]
+  },
+  {
+    "result": "44army",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_PISTOL",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "skills_required": [ "gun", 1 ],
+    "time": 20000,
+    "book_learn": [ [ "manual_pistol", 3 ], [ "recipe_bullets", 3 ] ],
+    "using": [ [ "ammo_bullet", 45 ] ],
+    "qualities": [ { "id": "CUT", "level": 1, "amount": 1 } ],
+    "components": [
+      [ [ "gunpowder", 160 ], [ "chem_black_powder", 320 ] ],
+      [ [ "lgpistol_primer", 25 ] ],
+      [ [ "bag_plastic", 1 ], [ "paper", 12 ] ]
```

Comment:
Though this does complicate making them individually, as we can not make it use half a paper or foil.